tez-mapreduce/src/main/java/org/apache/tez/mapreduce/output/MROutput.java

```diff
@@ -413,6 +414,7 @@ protected List<Event> initializeBase() throws IOException, InterruptedException
     }
     jobConf.setInt(MRJobConfig.APPLICATION_ATTEMPT_ID,
         getContext().getDAGAttemptNumber());
+    jobConf.set(MRJobConfig.MR_JOB_UUID, Utils.createJobUUID(getContext().getApplicationId().getClusterTimestamp(), getContext().getApplicationId().getId(), getContext().getDagIdentifier()));
```

Comment:
check for checkstyle violation, This might exceed the limit of character in a line.

Reply:
Addressed in update.